VMware docs update changes (#4256)
SumoLogic/sumologic-documentation: docs/integrations/containers-orchestration/vmware.md

## Prerequisites

> * **Supported Software Versions**. The VMware app works with vCenter and vSphere 6.5, 6.7, 7.0 and 8.0.

Reviewer: ```suggestion
* **Supported Software Versions**. The VMware app works with vCenter and vSphere 6.5, 6.7, 7.0, and 8.0.
```

To install and configure the Sumo Logic scripts, do the following:

1. On the server, host, or VM create a directory in which to put the Sumo Logic scripts from [Sumo Logic Scripts](https://github.com/SumoLogic/sumologic-vmware/tree/master/vsphere) for VMware. We recommend that you name the directory **/var/log/vmware* or something similar.
2. Download the Sumo Logic VMware scripts from [here](https://s3.amazonaws.com/appdevstore/VMWare/sumo-vsphere-ulm.zip), into the directory you just created.
3. Install [python](https://www.python.org/) version 3.6, or later.
4. Install [pyvmomi](https://pypi.org/project/pyvmomi/) 6.7.3:
   ```bash

>    pip install pyvmomi==6.7.3

Reviewer 2: @ankurch627 - We can point the user to this doc : https://github.com/vmware/pyvmomi?tab=readme-ov-file#compatibility-policy for choosing the version of this library based on there vsphere version

* **-o**: Remote vCenter Server port to use, default 443. Optional.
* **-u**: User name to use when connecting to vCenter server. Required.
* **-p**: Password to use when connecting to vCenter server. Required.
* **-f**: Output File Prefix. Target syslog server or file is required. Optional.
* **-ts**: Timestamp File. Default ‘.timelog_events’. Optional.

> * **-t**: Target Sumologic syslog server. Target syslog server or file is required. Optional.

Reviewer 2: This should be required ?

Reviewer 2: Sumologic installed collector host name. This is required in case of syslog server. If using local file source this parameter is not required.

> * **-t**: Sumologic installed collector host name. This is required in case of syslog server. If using local file source this parameter is not required.

Reviewer: ```suggestion
* **-t**: Host name of Sumo Logic installed collector. This is required in case of syslog server. If you are using local file source this parameter is not required.
```

  * **-u**: User name to use when connecting to vCenter server. Required.
  * **-p**: Password to use when connecting to vCenter server. Required.
  * **-ts**: Timestamp File. Default ‘.timelog_metrics’.

>   * **-t**: Host name where sumologic installed collector is installed. Required.

Reviewer: ```suggestion
  * **-t**: Host name where Sumo Logic installed collector is installed. Required.
```

>   * **-t**: Sumologic installed collector host name for streaming metrics. This is required in case of syslog server.

Reviewer 2: For metrics there is no syslog server required. Also in case of metric this is required. We can just write the line : 
"Host name where sumologic installed collector is installed. Required"

Author: Fixed this in the latest commit.

>   * **-t**: Sumologic installed collector host name for streaming metrics. This is required in case of syslog server.

Reviewer 2: ```suggestion
  * **-t**: Host name where sumologic installed collector is installed. Required
```

Author: Fixed this in the latest commit.
